POI.DiscordDotNet/Commands/Utils/CompareCommand.cs

```diff
@@ -51,7 +51,13 @@ public async Task Handle(CommandContext ctx, [RemainingText] string _)
 			var arguments = await ExtractArguments(ctx).ConfigureAwait(false);
 			if (arguments == null)
 			{
-				await _logger.LogError(ctx, "Something went wrong while trying to parse the arguments").ConfigureAwait(false);
+				var errorMessage = await ctx.RespondAsync("Try using the correct arguments, something like this: ```poinext cp <Player 1 (scoresaberId or mention)> <Player 2 (scoresaberId or mention, optional if you are linked)>```");
```

Comment:
Add `.ConfigureAwait(false)` at the end.
```suggestion
				var errorMessage = await ctx.RespondAsync("Try using the correct arguments, something like this: ```poinext cp <Player 1 (scoresaberId or mention)> <Player 2 (scoresaberId or mention, optional if you are linked)>```").ConfigureAwait(false);
```